### DISCUSSION TITLE
Modelling of a jet fan - unrealistically long air flow

### DISCUSSION THREAD

**Discussion by Z-0815** [firemodels/fds#13843] [Q&A]
Hello

i have a few questions about modelling a jet fan. I only want to test it in one room to check the throw range.

Mesh and Room:

# ---- Mesh ----
&MESH ID='MESH1', COLOR='YELLOW', IJK=254,64,38, XB=0,20, 7.5,12.5, 0,3, MPI_PROCESS=0 /
&MESH ID='MESH2', COLOR='YELLOW', IJK=127,32,19, XB=20,40, 7.5,12.5, 0,3, MPI_PROCESS=1 /
&MESH ID='MESH3', COLOR='YELLOW', IJK=127,48,19, XB=0,20, 0,7.5, 0,3, MPI_PROCESS=2 /
&MESH ID='MESH4', COLOR='YELLOW', IJK=127,48,19, XB=0,20, 12.5,20, 0,3, MPI_PROCESS=3 /
&MESH ID='MESH5', COLOR='YELLOW', IJK=127,48,19, XB=20,40, 0,7.5, 0,3, MPI_PROCESS=4 /
&MESH ID='MESH6', COLOR='YELLOW', IJK=127,48,19, XB=20,40, 12.5,20, 0,3, MPI_PROCESS=5 /
&MESH ID='MESH7', COLOR='YELLOW', IJK=127,32,19, XB=40,60, 7.5,12.5, 0,3, MPI_PROCESS=6 /
&MESH ID='MESH8', COLOR='YELLOW', IJK=127,48,19, XB=40,60, 0,7.5, 0,3, MPI_PROCESS=7 /
&MESH ID='MESH9', COLOR='YELLOW', IJK=127,48,19, XB=40,60, 12.5,20, 0,3, MPI_PROCESS=8 /

# ---- Open ----
&VENT ID='OPEN1', XB=0,0, 0,20, 0,3, SURF_ID='OPEN' /
&VENT ID='OPEN2', XB=60,60, 0,20, 0,3, SURF_ID='OPEN' /

1)	Would you model the jet with the downstream shroud or without it?
2)	I am modeling the jet as an HVAC duct with the exact same area as the real jet. Should a different approach be used?
Jet Ø355 – Mesh = 0.1573m
A= 0,3146m*0,3146m = 0,09897m²

# ---- Jet Fan ----
&OBST ID='JFAN1', COLOR='COLD GREY', XB=5,6.695, 9.843,10.1576, 2.5274,2,842 /
&VENT ID='JFAN1_vent_in', COLOR='BLUE', XB=5,5, 9.843,10.1576, 2.5274,2,842, SURF_ID='HVAC' /
&VENT ID='JFAN1_vent_out', COLOR='RED', XB=6.695,6.695, 9.843,10.1576, 2.5274,2,842, SURF_ID='HVAC',  /
&HVAC ID='JFAN1_hvac_in', TYPE_ID='NODE', DUCT_ID='JFAN1_hvac_duct', VENT_ID='JFAN1_vent_in' /
&HVAC ID='JFAN1_hvac_out', TYPE_ID='NODE', DUCT_ID='JFAN1_hvac_duct', VENT_ID='JFAN1_vent_out' /
&HVAC ID='JFAN1_hvac_duct', TYPE_ID='DUCT', NODE_ID='JFAN1_hvac_in','JFAN1_hvac_out', VOLUME_FLOW=2, DIAMETER=0.355 /

3)	What ceiling distance would you choose if the jet is positioned directly under the ceiling? One mesh size?
4)	FDS 6 uses Deardorff by default—would you stick with it or switch to "Dynamic Smagorinsky"?
5)	Is it possible to display the color bar logarithmically in Smokeview?
6)	What could be the reason if the jet does not lose energy?
I have the problem that the air jet is either too close to the ceiling and becomes too wide via the coanda effect. If it is deflected too far, the jet does not lose any energy. with a jet in a room, the jet area becomes longer and longer at 1m/s. Do you have any idea why this could be?

Br Z

**Comment by rmcdermo** [comment]
1. If you can afford the resolution, I would model the shroud.  You probably need 10 cells across the face of the jet.
2. HVAC should be good.
3. Not sure I understand this question.  I would set up the ceiling at the correct physical distance from the jet.  If you mean should you leave at leave one grid cell between the horizontal jet fan and the ceiling, then probably, maybe more if that actually matters for your case, you should test with and without.
4. See [fds/Validation/Turbulent_Jet](https://github.com/firemodels/fds/tree/master/Validation/Turbulent_Jet/FDS_Input_Files) (in [FDS_Verification_Guide](https://github.com/firemodels/fds/releases/download/FDS-6.9.1/FDS_Verification_Guide.pdf)) for the effects of turbulence models.  My opinion is use Deardorff.
5. I don't think this is very easy to do.  It's possible, but I think you'd have to define your own colorbar.  @gforney may weigh in here.
6. Could be resolution or inlet turbulence or not capturing the roughness elements of a real ceiling.

**Reply by Z-0815** [reply]
hi, 
thanks for the answer.

to 1. which mesh would you recommend:
Jet Ø355mm – A= 0,3146m*0,3146m = 0,09897m²
h/2 = 0,157mm
h/4 = 0,07865mm

to 6. what could i add to my code? i let the siumulation run for approx. 600 seconds. the natural openings at the back and front create an induction. the air flow is stretched.

here is the code, do you have an idea what i am doing wrong?
[v01_32.txt](https://github.com/user-attachments/files/18025516/v01_32.txt)

**Comment by rmcdermo** [comment]
I think you need more than 4 cells across your jet if you want to model the shear layer that controls how fast the jet decays.  10 to 20 cells is my recommendation.  I have debated this question with Dan Swenson at Thunderhead for years.  You need reasonable resolution to capture the physics of jet decay.  If you can't supply that resolution then the alternative is to specify the velocity field in the region of interest.  That is why we have the "VELOCITY PATCH" feature.  See 18.3.3 Specified Entrainment (Velocity Patch) in the user guide.  Take whatever jet field you like and specify it downstream of the fan.

**Reply by Z-0815** [reply]
thank you very much for your effort. i will try a mesh h/4, i.e. 16 cells distributed over the opening area. i still have to take a closer look at part 18.3.3. but i think my computing power should be sufficient for the fine mesh. 

i have a question about the distance to the ceiling. i would like to keep some distance, because the effect that the airflow becomes very wide (probably coanda effect) does not reflect what i want. should this effect also be better with the finer mesh?

**Reply by rmcdermo** [reply]
Think of things this way: Any length scale in the problem that you really need to resolve will require 10 cells across the *length*, not the area.  So, 4 cells across the fan is not enough.  Same goes for the space between the fan and the ceiling, if this distance is important, it needs to be resolved.  Beyond this, you need to make do with engineering approximations.  For example, you could do a back-of-the-envelop calculation for whatever boundary layer quantity you are interested in (heat transfer coefficient, etc.) and specify those directly instead of trying to resolve things with CFD.

**Reply by Z-0815** [reply]
ok, i see, that's a very fine mesh. does this have to be used over the entire expected length or just around the jet fan?

why is a mesh with h/2 or h/4 often recommended for garages? Similar problems with the air flow should occur here.

**Reply by rmcdermo** [reply]
The width of the jet changes with downstream distance from the exit.  Roughly speaking you need 10-20 cells across the width.

I don't know what recommendations you are referring to.  We certainly do not have any validation cases that use h/4 or less.  As I said, I've been debating this with folks for years.

**Reply by Z-0815** [reply]
Thank you for your explanation. I will try both options fine mesh and velocity patch.

**Reply by Z-0815** [reply]
since the velocity patch was mentioned as an alternative, what mesh size do you recommend here? 
i would design the velocity patch and then model an shroud around it, i am on the right track?


&PROP ID='p1', VELOCITY_COMPONENT=1, P0=20.2 /
&DEVC ID='t1', XYZ=0,0,0, QUANTITY='TIME', SETPOINT=0/
&DEVC XB=5,6.695, 9.8427,10.1573, 2.5254,2.84, QUANTITY='VELOCITY PATCH', PROP_ID='p1', DEVC_ID='t1'/ 

&OBST ID='Shroud Top', XB=5,6.695, 9.8427,10.1573, 2.84,2.84, SURF_ID='Duct'/ 
&OBST ID='Shroud Bottom', XB=5,6.695, 9.8427,10.1573, 2.5254,2.5254, SURF_ID='Duct'/
&OBST ID='Shroud Side -Y', XB=5,6.695, 9.8427,9.8427, 2.5254, 2.84, SURF_ID='Duct'/ 
&OBST ID='Shroud Side +Y', XB=5,6.695, 10.1573,10.1573, 2.5254, 2.84, SURF_ID='Duct'/

Should I use a deflector, how do I incorporate it into the code? via VELOCITY_COMPONENT=3, P0=-0.1?

**Reply by rmcdermo** [reply]
I would use your h/4 resolution.  You need a few cells just to see the profile.

The velocity component you want to specify is the one in the flow direction, which I would assume is x, so VELOCITY_COMPONENT=1.

I would first understand your field using Python or similar and make sure you understand how to specify the polynomial coefficients.
